test-coverage.txt

```diff
@@ -0,0 +1,56 @@
+ PASS  src/components/Card/Card.test.js
+ PASS  src/index.test.js (5.262s)
+ PASS  src/components/App/App.test.js (5.749s)
+ PASS  src/_test_/getYoutubeData.test.js
+ PASS  src/pages/NotFound/NotFound.test.js
+ PASS  src/components/Header/Header.test.js
+ PASS  src/components/Layout/Layout.test.js
+ PASS  src/components/Menu/Menu.test.js
+ PASS  src/components/LoginButton/Login.test.js
+ PASS  src/components/InputSearch/InputSearch.test.js
+ PASS  src/components/Toogle/ToogleTheme.test.js
+----------------------------|----------|----------|----------|----------|-------------------|
+File                        |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
+----------------------------|----------|----------|----------|----------|-------------------|
+All files                   |    93.75 |       50 |    83.33 |    93.75 |                   |
```

Comment:
The test coverage is good, the branch percentage is a little low but can be improved by adding more cases to the tests, if you need help ping me in slack